src/sentry/quotas/redis.py

```diff
@@ -68,17 +70,26 @@ def get_quotas(self, project):
                 window=oquota[1],
                 reason_code='org_quota',
             ),
-        )
+        ]
+        if key:
```

Comment:
I'm fine w/ keeping consistency of what we do here. If we REALLY need the memory savings we could avoid building them, but i dont think its a huge deal. This is about describing quotas as this stage vs enforcing them.